dockertest/environment.py

```diff
@@ -47,12 +55,7 @@ def get_selinux_context(path):
     When selinux is enabled, return the context of ``path``
     :param path: Full or relative path to a file or directory
     :return: SELinux context as a string or None if not set
+    :raises IOError: When file does not exist.
```

Comment:
Why single this one out for mentioning? We can't know what other exceptions selinux might raise; it's not really our job here to list them...